test/unittests/libp2p/net.cpp

```diff
@@ -52,292 +52,387 @@ BOOST_FIXTURE_TEST_SUITE(net, NetFixture)
+    }
 }
 
 BOOST_AUTO_TEST_CASE(neighboursPacket)
 {
-	if (test::Options::get().nonetwork)
-	{
-		clog << "Skipping test network/net/neighboursPacket. --nonetwork flag is set.\n";
-		return;
-	}
-
-	KeyPair k = KeyPair::create();
-	std::vector<std::pair<KeyPair,unsigned>> testNodes(TestNodeTable::createTestNodes(16));
-	bi::udp::endpoint to(boost::asio::ip::address::from_string("127.0.0.1"), 30000);
-
-	Neighbours out(to);
-	for (auto n: testNodes)
-	{
-		Node node(n.first.pub(), NodeIPEndpoint(boost::asio::ip::address::from_string("200.200.200.200"), n.second, n.second));
-		Neighbours::Neighbour neighbour(node);
-		out.neighbours.push_back(neighbour);
-	}
-	out.sign(k.secret());
-
-	bytesConstRef packet(out.data.data(), out.data.size());
-	auto in = DiscoveryDatagram::interpretUDP(to, packet);
-	int count = 0;
-	for (auto n: dynamic_cast<Neighbours&>(*in).neighbours)
-	{
-		BOOST_REQUIRE_EQUAL(testNodes[count].second, n.endpoint.udpPort);
-		BOOST_REQUIRE_EQUAL(testNodes[count].first.pub(), n.node);
-		BOOST_REQUIRE_EQUAL(sha3(testNodes[count].first.pub()), sha3(n.node));
-		count++;
-	}
+    if (test::Options::get().nonetwork)
+    {
+        clog << "Skipping test network/net/neighboursPacket. --nonetwork flag is set.\n";
+        return;
+    }
+
+    KeyPair k = KeyPair::create();
+    std::vector<std::pair<Public, unsigned>> testNodes(TestNodeTable::createTestNodes(16));
+    bi::udp::endpoint to(boost::asio::ip::address::from_string("127.0.0.1"), 30000);
+
+    Neighbours out(to);
+    for (auto n: testNodes)
+    {
+        Node node(n.first, NodeIPEndpoint(boost::asio::ip::address::from_string("200.200.200.200"),
+                               n.second, n.second));
+        Neighbours::Neighbour neighbour(node);
+        out.neighbours.push_back(neighbour);
+    }
+    out.sign(k.secret());
+
+    bytesConstRef packet(out.data.data(), out.data.size());
+    auto in = DiscoveryDatagram::interpretUDP(to, packet);
+    int count = 0;
+    for (auto n: dynamic_cast<Neighbours&>(*in).neighbours)
+    {
+        BOOST_REQUIRE_EQUAL(testNodes[count].second, n.endpoint.udpPort);
+        BOOST_REQUIRE_EQUAL(testNodes[count].first, n.node);
+        BOOST_REQUIRE_EQUAL(sha3(testNodes[count].first), sha3(n.node));
+        count++;
+    }
 }
 
 BOOST_AUTO_TEST_CASE(test_findnode_neighbours)
 {
-	// Executing findNode should result in a list which is serialized
-	// into Neighbours packet. Neighbours packet should then be deserialized
-	// into the same list of nearest nodes.
+    // Executing findNode should result in a list which is serialized
+    // into Neighbours packet. Neighbours packet should then be deserialized
+    // into the same list of nearest nodes.
 }
 
 BOOST_AUTO_TEST_CASE(kademlia)
 {
-	if (test::Options::get().nonetwork)
-	{
-		clog << "Skipping test network/net/kademlia. --nonetwork flag is set.\n";
-		return;
-	}
-
-	TestNodeTableHost node(8);
-	node.start();
-	node.setup();
```

Comment:
`setup()` and `populateAll()` generated a bunch of `NodeTable` instances, which doesn't seem to influence anything in the test